src/ pages/Login/Login.js

```diff
@@ -1,8 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Nav from '../../components/Nav';
 import './Login.scss';
 
 function Login() {
-  return <div>login</div>;
+  const [inputId, setinputId] = useState('');
+  const [inputPw, setinputPw] = useState('');
+  const [loginBtnActive, setloginBtnActive] = useState(false);
+  const inputIdEvent = e => setinputId(e.target.value);
+  const inputPwEvent = e => setinputPw(e.target.value);
+  const btnActive = () => {
+    setloginBtnActive(
+      inputId.includes('') && inputPw.length > 3 ? true : false
+    );
+  };
```

Comment:
```suggestion
  const [inputId, seInputId] = useState('');
  const [inputPw, setInputPw] = useState('');
  const [loginBtnActive, setLoginBtnActive] = useState(false);
  
  const inputIdEvent = e => setInputId(e.target.value);
  const inputPwEvent = e => setInputPw(e.target.value);
  
  const btnActive = () => {
    setloginBtnActive(inputId.includes('') && inputPw.length > 3);
  };
```
- 구분되는 요소들끼리는 한줄씩 띄워주세요
- 불리언 값 간단하게 써줄 수 있으면 간단하게 써주는게 좋겠습니다